ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-nat.j2

```diff
@@ -0,0 +1,108 @@
+{# doc - ip nat #}
+{% if ip_nat is arista.avd.defined %}
+
+## IP NAT
+{%     if ip_nat.kernel_buffer_size is arista.avd.defined %}
+
+| Settings | Value |
```

Comment:
I know the doc is not really uniform about this in the rest but it makes more sense to keep Setting singular
```suggestion
| Setting | Value |
```

Reply:
Agreed.

Reply:
Fixed in [ece7ab2](https://github.com/aristanetworks/ansible-avd/pull/2747/commits/ece7ab239d6f3915e0b9829b036a53e774724fce)